packages/block-library/src/breadcrumbs/index.php

```diff
@@ -435,10 +435,12 @@ function block_core_breadcrumbs_get_archive_breadcrumbs() {
 			$post_type = reset( $post_type );
 		}
 		$post_type_object = get_post_type_object( $post_type );
+		$title            = apply_filters( 'post_type_archive_title', $post_type_object->labels->name, $post_type );
```

Comment:
The passed value acts as a fallback and probably is better to have the same `$post_type_object->labels->archives` fallback.